Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift

```diff
@@ -280,6 +280,10 @@ class AssignmentDetailsViewController: UIViewController, AssignmentDetailsViewPr
 
     func chooseSubmissionType(_ types: [SubmissionType]) {
         let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        // set ipad properties to display modal
+        alert.popoverPresentationController?.sourceView = submitAssignmentButton?.superview
+        alert.popoverPresentationController?.sourceRect = CGRect(x: submitAssignmentButton?.superview?.frame.midX ?? 0, y: submitAssignmentButton?.superview?.frame.midY ?? 0, width: 0, height: 0)
```

Comment:
Using `CGRect.init(origin:size:)` could potentially clean this up. Not a blocker though.

```swift
alert.popoverPresentationController?.sourceRect = CGRect(origin: submitAssignmentButton?.superview?.frame.center ?? .zero, size: 0)
```

Reply:
hey that's a great idea, i like that